<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KHotKeys::Windowdef_simple_widget_ui</class>
  <widget class="QWidget" name="Windowdef_simple_widget_ui" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>588</width>
        <height>497</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QLabel" name="comment_label" >
          <property name="text" >
            <string>&amp;Comment:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>comment_lineedit</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="comment_lineedit" />
      </item>
      <item>
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="window_title_label" >
              <property name="text" >
                <string>Window &amp;title:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>window_title_lineedit</cstring>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="window_title_combo" >
              <item>
                <property name="text" >
                  <string>Is Not Important</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Contains</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Matches Regular Expression</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Contain</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is Not</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Match Regular Expression</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QLineEdit" name="window_title_lineedit" />
      </item>
      <item>
        <spacer name="Spacer5" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="window_class_label" >
              <property name="text" >
                <string>Window c&amp;lass:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>window_class_lineedit</cstring>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="window_class_combo" >
              <item>
                <property name="text" >
                  <string>Is Not Important</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Contains</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Matches Regular Expression</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Contain</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is Not</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Match Regular Expression</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QLineEdit" name="window_class_lineedit" />
      </item>
      <item>
        <spacer name="Spacer7" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="window_role_label" >
              <property name="text" >
                <string>Window &amp;role:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>window_role_lineedit</cstring>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="window_role_combo" >
              <item>
                <property name="text" >
                  <string>Is Not Important</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Contains</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Matches Regular Expression</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Contain</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Is Not</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Does Not Match Regular Expression</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QLineEdit" name="window_role_lineedit" />
      </item>
      <item>
        <spacer name="Spacer15" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer13" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="autodetect_button" >
              <property name="text" >
                <string>&amp;Autodetect</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer14" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="Spacer7_2" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="Q3GroupBox" name="window_types_groupbox" >
          <property name="title" >
            <string>Window Types</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="0" column="0" >
              <widget class="QCheckBox" name="type_normal_checkbox" >
                <property name="text" >
                  <string>Normal</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QCheckBox" name="type_dialog_checkbox" >
                <property name="text" >
                  <string>Dialog</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="QCheckBox" name="type_desktop_checkbox" >
                <property name="text" >
                  <string>Desktop</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QCheckBox" name="type_dock_checkbox" >
                <property name="text" >
                  <string>Dock</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>comment_lineedit</tabstop>
    <tabstop>window_title_combo</tabstop>
    <tabstop>window_title_lineedit</tabstop>
    <tabstop>window_class_combo</tabstop>
    <tabstop>window_class_lineedit</tabstop>
    <tabstop>window_role_combo</tabstop>
    <tabstop>window_role_lineedit</tabstop>
    <tabstop>autodetect_button</tabstop>
  </tabstops>
  <includes>
    <include location="global" >kdialog.h</include>
  </includes>
  <connections>
    <connection>
      <sender>window_class_combo</sender>
      <signal>activated(int)</signal>
      <receiver>Windowdef_simple_widget_ui</receiver>
      <slot>window_class_combo_changed(int)</slot>
    </connection>
    <connection>
      <sender>window_role_combo</sender>
      <signal>activated(int)</signal>
      <receiver>Windowdef_simple_widget_ui</receiver>
      <slot>window_role_combo_changed(int)</slot>
    </connection>
    <connection>
      <sender>window_title_combo</sender>
      <signal>activated(int)</signal>
      <receiver>Windowdef_simple_widget_ui</receiver>
      <slot>window_title_combo_changed(int)</slot>
    </connection>
  </connections>
</ui>
